server/src/main/scala/io/delta/kernelsharedtable/JsonPredicatePruner.scala

```diff
@@ -0,0 +1,172 @@
+/*
```

Comment:
I assume this is mostly copied and don't have much changes?

Reply:
yup, no changes